extras/rya.pcj.fluo/pcj.fluo.app/src/main/java/org/apache/rya/indexing/pcj/fluo/app/batch/JoinBatchInformation.java

```diff
@@ -1,4 +1,6 @@
 package org.apache.rya.indexing.pcj.fluo.app.batch;
+import static jline.internal.Preconditions.checkNotNull;
```

Comment:
This is probably supposed to be a different impl of this method.

Reply:
Done